docker-compose.yml

```diff
@@ -32,6 +32,17 @@ services:
       ACCEPT_EULA: Y
       SA_PASSWORD: MyS3cr3tPassw0rd
 
+  # Setup the runner
+  # we need to set the localhost and root password as they can be different
+  runner:
```

Comment:
I've found it clearer to name the service after the app, i.e. crawler.

Reply:
Hi Phil, they all seem to be prefixed by crawer, see screenshot:
![Screenshot 2021-01-05 at 14 37 30](https://user-images.githubusercontent.com/10407809/103658818-a1369e80-4f63-11eb-8f59-08e1420ba04f.png)

So if changed to 'crawler' you get 'crawler_crawler'

Reply:
Yip, that's true, root name is used to prepend all the services in the "stack" but imagine you add another service (in a larger stack) to the stack which behaves in a similar manner, calling it by it's name instead of how/what it does is clearer. I think I'm arguing from a consistence point of view because that is how other stacks are defined.

Reply:
OK no probs